src/main/kotlin/domain/Players.kt

```diff
@@ -0,0 +1,8 @@
+package domain
+
+class Players (
+    val dealer : Dealer,
+    val users: List<User>
+){
+
+}
```

Comment:
```suggestion
)
```
불필요한 중괄호는 제거해보면 어떨까요?